Source/WebCore/inspector/agents/InspectorMemoryAgent.cpp

```diff
@@ -164,12 +168,14 @@ void InspectorMemoryAgent::collectSample(const ResourceUsageData& data)
     categories->addItem(WTFMove(pageCategory));
     categories->addItem(WTFMove(otherCategory));
 
-    auto event = Inspector::Protocol::Memory::Event::create()
-        .setTimestamp(m_environment.executionStopwatch().elapsedTimeSince(data.timestamp).seconds())
-        .setCategories(WTFMove(categories))
-        .release();
+    if (CheckedPtr environment = m_environment.get()) {
```

Comment:
We don't expect environment to be null. So, in most cases, I prefer using early exit `if (!environment) return { }` or similar, because it reduces code indentation and it can be turned into a macro easily if we need to perform multiple WeakPtr validations. See macros in `WebAutomationSession.h`.